### PR TITLE
Adds new, more lore relevant names to station naming

### PR DESCRIPTION
**Pull request by Yellow-Mushroom**
## About the PR 

Adds:
-New category for roundstart station naming (Frontier Locations)
-New categories for captain station naming (Frontier Bodies, Frontier Locations, Sol Bodies, Organizations)
-Miscellanious lore names/relevant names to the station namers (such as: supply, FAAE, NanoTrasen, Regional, etc)
-Gives Oshan a set roundstart name (Oshan Laboratory, Abzu) 

Removes: 
-Some more military/prison labor related names from the roundstart station namer (these are not removed from the captain's station namer,  so you can still name the station "Clown Prison")
-Sol System names (such as earth/mars/venus) from the roundstart station name (still in cap's station namer)

Changes:
-Changes the composition of station naming, and at what rate the station names are chosen, in order to create a less word salady station name
-General typo's/miscapitalization
-Reorganizes which categories some of the names are located in order to better fit round start station naming
-Fixes Horizon and Atlas not being labeled as Ships in proc/station_or_ship

## Why's this needed? 

Our station namer isn't really accurate with current game lore, and can lead to a lot of players thinking that the station is currently in the Sol System with how often Earth and Venus orbitals occured. This further solidify's newer coglore by allowing for "Regina Orbitals" and what not. 

Having Oshan have a set name 1) stops it from getting roundstart random names such as "extraplanetary/orbital/asteroid", and 2) places it on Abzu, with many new players being confused on where Oshan is if not Earth due to the water and fish

This also allows for Frontier Organizations (such as Hafgan Heavy Industry, Xiang Geisel, and others) to have the station traded to them because it happens a surprising amount on RP, 

It also helps with word salad/ridiculously long names. Theyre still possible but a lot more uncommon

### CODE REVIEW

**Comment by Vaskritaya** [comment]
Some notes I have after taking the time to actually look at the options for once in my life as a result of this pr

In the Countries section, Kosovo and Uzbekistan are both listed twice. Additionally, Countries contains Naughty, presumably as a counter to Nice (an actual location) but it's still a little odd.

Much more subjectively, I feel like "Ethno" and "Schizo" could afford to be removed from the "language prefixes", since i cant see any particularly good use for them and it seems like Goon has been slowly trying to move away from things like the latter of the two. A similar case could be made for "Eighty-eight" from the numbers category, which is probably actually intended as a back to the future reference, but that's just nitpicking.

**Comment by Yellow-Mushroom** [comment]
> Some notes I have after taking the time to actually look at the options for once in my life as a result of this pr
> 
> In the Countries section, Kosovo and Uzbekistan are both listed twice. Additionally, Countries contains Naughty, presumably as a counter to Nice (an actual location) but it's still a little odd.
> 
> Much more subjectively, I feel like "Ethno" and "Schizo" could afford to be removed from the "language prefixes", since i cant see any particularly good use for them and it seems like Goon has been slowly trying to move away from things like the latter of the two. A similar case could be made for "Eighty-eight" from the numbers category, which is probably actually intended as a back to the future reference, but that's just nitpicking.

YIKES YEAH NO ILL DEF GET THOSE RN

File: strings/station_name_whitelist.txt [hunk 1]
```diff
@@ -10,13 +10,13 @@ solBodies@=Earth@,Mercury@,Venus@,Mars@,Jupiter@,Saturn@,Uranus@,Neptune@,Ceres@
...
...Cobrastan@,Colombia@,Comoros@,Congo@,Croatia@,Cuba@,Cyprus@,Denmark@,Djibouti@,Dominica@,Ecuador@,Egypt@,England@,Eritrea@,Estonia@,Ethiopia@,Fiji@,Finland@,France@,Gabon@,Gambia@,Georgia@,Germany@,Ghana@,Greece@,Grenada@,Guadeloupe@,Guatemala@,Guinea@,Guyana@,Haiti@,Honduras@,Hungary@,Iceland@,India@,Indonesia@,Iran@,Iraq@,Ireland@,Israel@,Italy@,Jamaica@,Japan@,Jordan@,Kazakhstan@,Kenya@,Kolechia@,Korea@,Kosovo@,Kuwait@,Kyrgyz@,Laos@,Latvia@,Lebanon@,Lesotho@,Liberia@,Libya@,Liechtenstein@,Lithuania@,Luxembourg@,Macedonia@,Madagascar@,Malawi@,Malaysia@,Maldives@,Mali@,Malta@,Martinique@,Mauritania@,Mauritius@,Mayotte@,Mexico@,Moldova@,Monaco@,Mongolia@,Montenegro@,Montserrat@,Morocco@,Mozambique@,Namibia@,Naughty@,Nepal@,Netherlands@,Nicaragua@,Nice@,Nigeria@,Norway@,Oman@,Pakistan@,Panama@,Paraguay@,Peru@,Philippines@,Poland@,Portugal@,Qatar@,Reunion@,Romania@,Rwanda@,Samoa@,Scotland@,Senegal@,Serbia@,Seychelles@,Singapore@,Slovenia@,Somalia@,Spain@,Sudan@,Suriname@,Swaziland@,Sweden@,Switzerland@,Syria@,Tajikistan@,Tanzania@,Thailand@,Togo@,Tunisia@,Turkey@,Turkmenistan@,UAE@,Uganda@,Ukraine@,Uruguay@,USA@,Uzbekistan@,Venezuela@,Vietnam@,Wales@,Yemen@,Zambia@,Zanzibar@,Zimbabwe
```
Comment (ZeWaka) [inline strings/station_name_whitelist.txt:13]: why is Nice even on here as a country? It's a city lol

File: strings/station_name_whitelist.txt [hunk 2]
```diff
@@ -1,14 +1,22 @@
...
```
Comment (ZeWaka) [inline strings/station_name_whitelist.txt:13]:
```suggestion
countries@=Afghanistan@,Albania@,Algeria@,Andorra@,Angola@,Anguilla@,Argentina@,Armenia@,Arstotzka@,Australia@,Austria@,Azerbaijan@,Bahamas@,Bahrain@,Bangladesh@,Barbados@,Belarus@,Belgium@,Belize@,Benin@,Bermuda@,Bhutan@,Bolivia@,Botswana@,Brazil@,Brexit@,Bulgaria@,Burundi@,Cambodia@,Cameroon@,Canada@,Catmandu@,Chad@,Chile@,China@,Cobrastan@,Colombia@,Comoros@,Congo@,Croatia@,Cuba@,Cyprus@,Denmark@,Djibouti@,Dominica@,Ecuador@,Egypt@,England@,Eritrea@,Estonia@,Ethiopia@,Fiji@,Finland@,France@,Gabon@,Gambia@,Georgia@,Germany@,Ghana@,Greece@,Grenada@,Guadeloupe@,Guatemala@,Guinea@,Guyana@,Haiti@,Honduras@,Hungary@,Iceland@,India@,Indonesia@,Iran@,Iraq@,Ireland@,Israel@,Italy@,Jamaica@,Japan@,Jordan@,Kazakhstan@,Kenya@,Kolechia@,Korea@,Kosovo@,Kuwait@,Kyrgyz@,Laos@,Latvia@,Lebanon@,Lesotho@,Liberia@,Libya@,Liechtenstein@,Lithuania@,Luxembourg@,Macedonia@,Madagascar@,Malawi@,Malaysia@,Maldives@,Mali@,Malta@,Martinique@,Mauritania@,Mauritius@,Mayotte@,Mexico@,Moldova@,Monaco@,Mongolia@,Montenegro@,Montserrat@,Morocco@,Mozambique@,Namibia@,Naughty@,Nepal@,Netherlands@,Nicaragua@,Nigeria@,Norway@,Oman@,Pakistan@,Panama@,Paraguay@,Peru@,Philippines@,Poland@,Portugal@,Qatar@,Reunion@,Romania@,Rwanda@,Samoa@,Scotland@,Senegal@,Serbia@,Seychelles@,Singapore@,Slovenia@,Somalia@,Spain@,Sudan@,Suriname@,Swaziland@,Sweden@,Switzerland@,Syria@,Tajikistan@,Tanzania@,Thailand@,Togo@,Tunisia@,Turkey@,Turkmenistan@,UAE@,Uganda@,Ukraine@,Uruguay@,USA@,Uzbekistan@,Venezuela@,Vietnam@,Wales@,Yemen@,Zambia@,Zanzibar@,Zimbabwe
```

**Pull request merged**